File: URI-py/1048.py
# -*- coding: utf-8 -*-
s = float(input())
if s >= 0 and s <= 400:
    s1 = s*1.15
    r1 = s*.15
    p1 = ("15 %")
    print("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %s"%(s1,r1,p1))
elif s > 400 and s <= 800:
    s2 = s*1.12
    r2 = s*.12
    p2 = ("12 %")
    print("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %s"%(s2,r2,p2))
elif s >= 800.01 and s <= 1200:
    s3 = s*1.1
    r3 = s*.1
    p3 = ("10 %")
    print("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %s"%(s3,r3,p3))
elif s >= 1200.01 and s <= 2000:
    s4 = s*1.07
    r4 = s*.07
    p4 = ("7 %")
    print("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %s"%(s4,r4,p4))
else:
    s5 = s*1.04
    r5 = s*.04
    p5 = ("4 %")
    print("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %s"%(s5,r5,p5))

File: URI-lua/1048.lua
salary = io.read("*n")
if salary <= 400 then
  ganho = salary*.15
  bonus = 15
  salary = salary+ganho
elseif salary > 400 and salary <= 800 then
  ganho = salary*.12
  bonus = 12
  salary = salary+ganho
elseif salary > 800 and salary <= 1200 then
  ganho = salary*.10
  bonus = 10
  salary = salary+ganho
elseif salary > 1200 and salary <= 2000 then
  ganho = salary*.07
  bonus = 7
  salary = salary+ganho
elseif salary > 2000 then
  ganho = salary*.04
  bonus = 4
  salary = salary+ganho
end
io.write(string.format("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %d %%\n",salary,ganho,bonus))
